Dockerfile

```diff
@@ -37,6 +38,8 @@ RUN set -x \
         --delete               "Server/Service/Engine/Host/Context/@debug" \
                                "${CONF_INSTALL}/conf/server.xml"
 
+ADD start.sh ${CONF_INSTALL}/bin/
```

Comment:
I would like to see it more inline with the implementations of other official images like postgres for an example, using the entrypoint script name `docker-entrypoint.sh` copied to the root instead.

```
COPY docker-entrypoint.sh /

ENTRYPOINT ["/docker-entrypoint.sh"]
```